agent/lib/pbench/add_metalog_option.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+# Usage: pbench-add-metalog-option <metadata log file> <section> <option>
```

Comment:
Does this module "become" the `pbench-add-metalog-option` CLI interface?

Reply:
That's the intention yes. This can be renamed to that or we can have the command aliased or create a dynamic link.